common/datasets/librispeech.py

```diff
@@ -354,6 +354,50 @@ def get_special_lemma_lexicon(add_unknown_phoneme_and_mapping=True):
     return lex
 
 
+@lru_cache()
+def _get_raw_bliss_lexicon(
+    use_stress_marker,
```

Comment:
Is there a reasonable default for this bool? Or is this intentionally not set? `get_bliss_lexicon` has `False` as default.

Reply:
If you want to use this private function, you should think about what you want to do. so I think there is no reasonable default, IMHO